x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/260_rule_query_search.yml

```diff
@@ -236,4 +236,151 @@ setup:
   - match: { hits.total.value: 1 }
   - match: { hits.hits.0._id: 'doc1' }
 
+---
+"Perform a rule query with an organic query that must be rewritten to another query type":
+
```

Comment:
My guess is that this needs to have a `skip` version for the version that this is fixed in. Generally, how to handle this, is to make it restrictive on main (e.g. 8.13 only) (this may be even trickier if 8.13 branch is already cut, if so, this skip should initially be 8.14, and after backports, adjust). and in the back port adjust it to `8.12.2/3`. Once the backport is merged, adjust main to `8.12.2/3`.

Reply:
Thanks for this - I'll make sure to do this